packages/mdx/mdx.1.0.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis: "Executable markdown files"
+description: """
+`mdx` allows to execute code blocks inside markdow files.
+
+### Supported extensions
+
+#### Cram tests
+
+Codes blocks using `sh` are considered as cram tests:
+
+```sh
+  $ for i in `seq 1 3`; do echo $i; done
+  1
+  2
+  3
+```
+
+which can be executed with `mdx test <file.md>`
+
+#### OCaml code
+
+Code blocks using `ocaml` are considered either normal OCaml files
+or toplevel fragements"""
+maintainer: "Thomas Gazagnaire <thomas@gazagnaire.org>"
+authors: "Thomas Gazagnaire <thomas@gazagnaire.org"
+license: "ISC"
+homepage: "https://github.com/realworldocaml/mdx"
+doc: "https://realworldocaml.github.io/mdx/"
+bug-reports: "https://github.com/realworldocaml/mdx/issues"
+depends: [
+  "dune" {build}
+  "fmt"
+  "astring"
+  "logs"
+  "cmdliner"
+  "re" {>= "1.7.2"}
+  "ppx_tools"
+  "lwt" {test}
+]
+available: "ocaml-version" >= "4.06.0"
```

Comment:
I am not sure this is a valid opam2 field. /cc @rjbou and @AltGr to check.

Reply:
You are right. Opam2 requires a bound on ocaml in the deps field